.idea/codeStyles/Project.xml

```diff
@@ -1,18 +0,0 @@
-<component name="ProjectCodeStyleConfiguration">

```

Comment:
Huh, I wonder how it originally got in, I thought we gitignored idea files

Reply:
Yeah, we have it in `.gitignore` but `gitignore` does not work for files already added to the index. I've personally mixed feeling about this being in app `.gitignore` - if developer is using Rubymine than they should have this in their global `.gitignore` file instead.